signature-type.go

```diff
@@ -27,6 +27,8 @@ const (
 	SignatureV4Streaming
 )
 
+var emptySHA256 = sum256([]byte{})
```

Comment:
I copy/pasted it from the previous code:
`shaHeader = hex.EncodeToString(sum256([]byte{}))`

let me fix that.